### PR TITLE
feat(docs): add k8s minimal rbac

### PR DESCRIPTION
**Pull request by ethanfrogers**
add minimal rbac role for kubernetes access.

@lwander PTAL. it might be nicer if we just reference the section in the V1 provider to prevent duplication. thoughts?

also, i'll take care of trailing whitespace changes.

### CODE REVIEW

**Review by lwander — APPROVED** [review]
Thanks! I appreciate it :)

File: setup/providers/kubernetes-v2/index.md [hunk 1]
```diff
@@ -41,6 +41,59 @@ The Kubernetes provider has two requirements:
...
+  resources: ["pods", "services", "secrets"]
+  verbs: ["*"]
+- apiGroups: ["autoscaling"]
+  resources: ["horizontalpodautoscalers"]
+  verbs: ["list", "get"]
+- apiGroups: [“apps”]
+  resources: [“controllerrevisions”, "statefulsets"]
+  verbs: [“list”]
+- apiGroups: ["extensions", "app"]
+  resources: ["deployments", "replicasets", "ingresses", "daemonsets"]
```
Comment (davidxia) [inline setup/providers/kubernetes-v2/index.md:74]: @ethanfrogers Just curious: does Spinnaker need mutating permissions on daemonsets? When does it change them?

Comment (davidxia) [reply setup/providers/kubernetes-v2/index.md:74]: I guess the [KubernetesDaemonSetHandler](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/deployer/KubernetesDaemonSetHandler.java) needs it?

Comment (lwander) [reply setup/providers/kubernetes-v2/index.md:74]: Yup, you can effectively `kubectl apply` daemonset changes, and have Kubernetes roll them out.

**Pull request merged**